.gitignore

```diff
@@ -20,7 +20,6 @@ yarn-error.log
 ###> symfony/framework-bundle ###
 /.env
 /public/bundles/
-/var/
```

Comment:
`/var/*` is already excluded a few lines before (but with some other rules afterwards). This one overrides the work done above.